tests/client/components/DropDownModule.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import DropDownModule from '../../../client/components/DropDownModule'
+
+test('<DropDownModule/>', () => {
```

Comment:
Did you run these tests? - At the moment, these tests won't work. You may need to look at the file name, compare it to the others. There's something you're missing for it to run properly